runtime/tests/vm/resolvefield_tests.c

```diff
@@ -149,6 +149,14 @@ createFakeROMClass(U_8 *buffer, UDATA bufferSize, const char *className, testFie
 #undef BUF_ALLOC
 }
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+static J9ClassLoader *
+createFakeClassLoader() {
```

Comment:
What's the purpose of this?

Reply:
I added a `J9ClassLoader *classLoader` parameter to the `fieldOffsetsStartDo()` method in `resolvefield.c`. I intended to write a `createFakeClassLoader()` method to go along with this change for`fieldOffsetsStartDo()` tests in `resolvefield_tests.c`.